src/StackExchange.Redis.Extensions.Core/Implementations/RedisCacheConnectionPoolManager.cs

```diff
@@ -1,65 +1,135 @@
-﻿using System;
-using System.Collections.Concurrent;
-using System.Linq;
-using StackExchange.Redis.Extensions.Core.Abstractions;
-using StackExchange.Redis.Extensions.Core.Configuration;
-
-namespace StackExchange.Redis.Extensions.Core.Implementations
+﻿namespace StackExchange.Redis.Extensions.Core.Implementations
 {
-	public class RedisCacheConnectionPoolManager : IRedisCacheConnectionPoolManager
-	{
-		private static ConcurrentBag<Lazy<ConnectionMultiplexer>> connections;
-		private readonly RedisConfiguration redisConfiguration;
-
-		public RedisCacheConnectionPoolManager(RedisConfiguration redisConfiguration)
-		{
-			this.redisConfiguration =  redisConfiguration;
-			Initialize();
-		}
-
-		public void Dispose()
-		{
-			var activeConnections = connections.Where(lazy => lazy.IsValueCreated).ToList();
-
-			foreach (var connection in activeConnections)
-			{
-				connection.Value.Dispose();
-			}
-
-			Initialize();
-		}
-
-		public IConnectionMultiplexer GetConnection()
+    using System;
```

Comment:
Does it make sense to move all usings under namespace? Project itself do not use this approach, so it seems that such change is inconsistent

Reply:
I'll do it into the v6 branch

Reply:
> Does it make sense to move all usings under namespace? Project itself do not use this approach, so it seems that such change is inconsistent

My bad. Actually, it does not make such a difference; I accidentally applied code-cleanup in Rider, but I totally agree with you that it is not consistent with the project. I will fix this...